README.md

```diff
@@ -16,7 +16,7 @@ internationalized software.
 In the US and a handful of other countries, temperature is measured in degrees
 Fahrenheit. The rest of the world uses degrees Celsius. Some countries measure
 distances and lengths in miles, yards or feet, and inches, others use
-kilometers, meters, and centimeters. The Scandinavian Mile is a common unit of
+kilometers, meters, and entimeters. The Scandinavian Mile is a common unit of
```

Comment:
Why did this change?

Reply:
> We should wait for approval from both TG1 (the meeting yesterday) and TG2 (the monthly ECMA-402 meeting). We have TG1 approval, but not TG2. So, please don't merge this PR until we have sign-off at next week's TG2 meeting.

We have received the approval a moment ago :)

Reply:
Done !